setup.py

```diff
@@ -73,7 +73,7 @@
     packages=[package for package in find_packages() if package.startswith("stable_baselines3")],
     package_data={"stable_baselines3": ["py.typed", "version.txt"]},
     install_requires=[
-        "gym>=0.17,<0.20",  # gym 0.20 breaks atari-py behavior
+        "gym>=0.21",  # Remember to also update gym version in "extra" below when this changes
```

Comment:
I would probably fixed the version until https://github.com/DLR-RM/stable-baselines3/pull/780 is ready